test/lib/utils/group-helper.test.js

```diff
@@ -0,0 +1,52 @@
+const { parse, sep } = require('path');
+const GroupHelper = require('../../../lib/utils/group-helper');
+
+describe('group-helper tests', () => {
+    describe('resolveFilePath', () => {
+        it('should return correct path : sample/entry.js', done => {
+            const GroupHelperOptions = [{ entry: 'test/lib/utils/sample/entry.js' }, { progress: 'bar' }, { sourcemap: 'eval' }, { dev: false }, { prod: true }];
+            const GroupHelperObj = new GroupHelper(GroupHelperOptions);
+            const entryPath = GroupHelperObj.resolveFilePath(GroupHelperOptions[0].entry);
+            const { dir, base } = parse(entryPath);
```

Comment:
Use `afterEach` to extract the dir and base, if possible

Reply:
`afterEach` of jest ? 
I didn't get how it will work! You want to use this line
```js
const { dir, base } = parse(entryPath);
```
in `afterEach` ?
Then how'd I take those values inside the test cases?

Reply:
Declare them as global variables and reassign them, so we can reduce duplicate code 👍